### DISCUSSION TITLE
All the preprints have been claimed - what is my next step?

### DISCUSSION THREAD

**Discussion by DrMattG** [daniel1noble/ecoevo_1000#6] [Q&A]
It looks like all the preprints have been claimed - should I start "stepping on toes" and do some of the preprints not yet extracted OR should I start to check some?

**Comment by mlagisz** [comment]
Hi Matt, please wait for now - we will prepare data cross-checking instructions once all extractions are completed.

**Reply by shafieisabets** [reply]
Cool Losia! Please also let me know when the next steps will start as I would like to do work and contribute to the data cross-checking stage.
